package.json

```diff
@@ -123,9 +123,10 @@
     "postinstall-operator": "rimraf templates/che-operator && cpx 'node_modules/eclipse-che-operator/deploy/**' 'templates/che-operator'",
     "postinstall-repositories": "yarn upgrade eclipse-che eclipse-che-operator eclipse-che-minishift",
     "postinstall-cleanup": "rimraf node_modules/eclipse-che && rimraf node_modules/eclipse-che-operator && rimraf node_modules/eclipse-che-minishift",
-    "test": "jest",
```

Comment:
I think this command is executed also in other tests not only in e2e.

Reply:
Sorry, I accidentally remove it. I will recover it.